DirectProgramming/C++SYCL_FPGA/Tutorials/Features/multi_aocx/device_code_split/src/simple.cpp

```diff
@@ -0,0 +1,84 @@
+#include <sycl/sycl.hpp>
```

Comment:
If many of the cpp files are similar, can we put them in a common location that all the sub-tests can access?